tests/typecheck/views/decorators/test_csrf.yml

```diff
@@ -0,0 +1,15 @@
+-   case: view_decorator
+    main: |
+      from django.http.request import HttpRequest
+      from django.http.response import HttpResponse
+      from django.views.decorators.csrf import csrf_protect
+
+      @csrf_protect
+      def good_view(request: HttpRequest) -> HttpResponse:
```

Comment:
Add a test case with async view as well, please.

Reply:
And add a test case with additional parameters, e.g. `def good_view2(request: HttpRequest, url_param: str) -> HttpResponse`